Jenkinsfile

```diff
@@ -177,7 +177,6 @@ pipeline {
 
                             # 1st-party
                             build-server-rpm percona-dashboards grafana-dashboards
-                            build-server-rpm pmm-manage
```

Comment:
@hors why we don't call `build/bin/build-server` there?

Reply:
@AlekSi  I am not 100% sure but I think it was done in order to split one big build into a few stages because 'build-server' builds pmm-client, pmm-server rpms and pmm-server docker. Also we can ask @delgod  about it.